sources/updater/updog/src/main.rs

```diff
@@ -479,12 +514,12 @@ fn main_inner() -> Result<()> {
                 &manifest,
                 &current_release.version_id,
                 &variant,
-                arguments.force_version,
+                get_target_version(&config, &arguments.force_version)?,
```

Comment:
Maybe save the result of this as a variable too, since we're calling/checking this multiple times?